### DISCUSSION TITLE
Seeing strange alerts

### DISCUSSION THREAD

**Discussion by markm68k** [jasonacox/Powerwall-Dashboard#323] [Q&A]
I installed **Powerwall Dashboard** for the first time about a month ago and am seeing some alerts that I don't understand. See the attached screenshot.

Specifically, what do **PINV_a043_gridResistanceTooHigh**, **PINV_a067_overvoltageNeutralChassis**, and **POD_w110_SW_EOC** mean?

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/45903780/decd6389-04bc-46a2-8b50-698bcbc5e624)

**Comment by jasonacox** [comment]
Hi @mark-miller0 ! 

We have been archiving these alerts as they show up, and adding details as we discover them.  You can check out the [Devices and Alerts list here](https://github.com/jasonacox/pypowerwall#devices-and-alerts) on the pypowerwall.com page.

* PINV_a043_gridResistanceTooHigh - I haven't seen this one before - The PINV prefix means this is coming from the Inverter (as you probably already guessed). From your graphic, this seems to be intermittent and not nominal. Do you see any odd behavior? It seems to be happening at night when your battery is in discharge mode or you are on the grid.
* PINV_a067_overvoltageNeutralChassis - It's not clear what this means, but it seems to be nominal (it appears on working systems).
* POD_w110_SW_EOC - This is "End of Charge" This triggers when full backfeed starts and battery is at 100%.

**Reply by markm68k** [reply]
Thank you @jasonacox !

I am going to follow up with my installer and see what they say about the PINV alerts and will report back here with any new information.

**Reply by markm68k** [reply]
> Do you see any odd behavior? It seems to be happening at night when your battery is in discharge mode or you are on the grid.

I don’t see any odd behavior but wouldn’t know what to look for.

**Reply by jasonacox** [reply]
I would look at the Energy Usage graph.  Potential odd behavior would be any unusual spikes or flapping between Powerwall discharge and grid usage, especially during the alerts.
